### DISCUSSION TITLE
Kwikset 910 lock won't stay joined

### DISCUSSION THREAD

**Discussion by ejbrownie** [Koenkk/zigbee2mqtt#12396] [Devices]
I have a Kwikset 910 that I have removed from the previous Zigbee network (pressed the B button 9 times). When I try to join Z2Mqtt sees it join and then immediately says it has left the network. I am running a Sonoff stick flashed to the latest firmware. The logs are as follows:

Info 2022-05-05 19:58:55MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"910 SmartCode traditional electronic deadbolt","exposes":[{"features":[{"access":7,"description":"State of the lock","name":"state","property":"state","type":"binary","value_off":"UNLOCK","value_on":"LOCK"},{"access":1,"description":"Actual state of the lock","name":"lock_state","property":"lock_state","type":"enum","values":["not_fully_locked","locked","unlocked"]}],"type":"lock"},{"access":1,"description":"Remaining battery in %","name":"battery","property":"battery","type":"numeric","unit":"%","value_max":100,"value_min":0},{"features":[{"access":2,"description":"User ID to set or clear the pincode for","name":"user","property":"user","type":"numeric"},{"access":2,"description":"Type of user, unrestricted: owner (default), (year|week)_day_schedule: user has ability to open lock based on specific time period, master: user has ability to both program and operate the door lock, non_access: user is recognized by the lock but does not have the ability to open the lock","name":"user_type","property":"user_type","type":"enum","values":["unrestricted","year_day_schedule","week_day_schedule","master","non_access"]},{"access":2,"description":"Whether the user is enabled/disabled","name":"user_enabled","property":"user_enabled","type":"binary","value_off":false,"value_on":true},{"access":2,"description":"Pincode to set, set pincode to null to clear","name":"pin_code","property":"pin_code","type":"numeric"}],"name":"pin_code","property":"pin_code","type":"composite"},{"access":1,"description":"Triggered action on the lock","name":"action","property":"action","type":"enum","values":["unknown","lock","unlock","lock_failure_invalid_pin_or_id","lock_failure_invalid_schedule","unlock_failure_invalid_pin_or_id","unlock_failure_invalid_schedule","one_touch_lock","key_lock","key_unlock","auto_lock","schedule_lock","schedule_unlock","manual_lock","manual_unlock","non_access_user_operational_event"]},{"access":1,"description":"Source of the triggered action on the lock","name":"action_source_name","property":"action_source_name","type":"enum","values":["keypad","rfid","manual","rf"]},{"access":1,"description":"ID of user that triggered the action on the lock","name":"action_source_user","property":"action_source_user","type":"numeric"},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"99100-045","options":[{"access":2,"description":"Expose pin of this lock in the published payload (default false).","name":"expose_pin","property":"expose_pin","type":"binary","value_off":false,"value_on":true}],"supports_ota":false,"vendor":"Kwikset"},"friendly_name":"0x002446fffd0872bb","ieee_address":"0x002446fffd0872bb","status":"successful","supported":true},"type":"device_interview"}'
Info 2022-05-05 19:58:55Configuring '0x002446fffd0872bb'
Info 2022-05-05 19:58:56MQTT publish: topic 'zigbee2mqtt/0x002446fffd0872bb', payload '{"action":null,"action_source_name":null,"action_source_user":null,"battery":70,"linkquality":83,"state":null}'
Info 2022-05-05 19:58:56Successfully configured '0x002446fffd0872bb'
Warning 2022-05-05 19:59:01Device '0x002446fffd0872bb' left the network
Info 2022-05-05 19:59:01MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x002446fffd0872bb","ieee_address":"0x002446fffd0872bb"},"type":"device_leave"}'

**Comment by Ryan-Langley** [comment]
Have the same exact problem.  Find a solution?

**Comment by kernelheap** [comment]
Facing similar issues with kwikset 910 Zigbee model. Device leaves the network exactly 5 seconds after "Successfully configured message". Tried latest coordinator firmware update of Sonoff but no luck. Has anyone been successfully able to use Kwikset 910 with Sonoff Zigbee USB dongle.
